dev/operator_local_debugger.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+arg1=${1:-""}
+
+provider=""
+if [ "$arg1" = "--aws" ]; then
+  provider="aws"
+elif [ "$arg1" = "--gcp" ]; then
+  provider="gcp"
+else
+  echo "provider must be set: either pass in the --aws or the --gcp flag"
+  exit 1
+fi
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. >/dev/null && pwd)"
+DEBUG_CMD="dlv --listen=:2345 --headless=true --api-version=2 debug $ROOT/pkg/operator"
```

Comment:
Should we add some dev instructions (for us) on installing delve? Or even better yet, install the package with the `tools` command in the Makefile.